docs/specification.md

```diff
@@ -1,4 +1,4 @@
-# Version 0.2
+# Version 0.3
```

Comment:
```suggestion
# Version 0.2
```

let's leave this version at 0.2 (if you really want to, increment to 0.2.1)

Reply:
I just prefer to bump the version in some way when making backwards incompatible changes